results/intfloat__multilingual-e5-small/CzechSoMeSentimentClassification.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dataset_revision": "6ced1d87a030915822b087bf539e6d5c658f1988",
+  "mteb_dataset_name": "CzechSoMeSentimentClassification",
+  "mteb_version": "1.8.3",
+  "test": {
+    "accuracy": 0.5586,
```

Comment:
Not very good performance on either of the models on this one.
Any ideas why?
Maybe we could try bumping up the number of samples per class for training or using kNN instead of LogReg